sdk/tests/platform/test_az_mqtt5_policy.c

```diff
@@ -93,6 +94,8 @@ static az_result test_inbound_hfsm_root(az_event_policy* me, az_event event)
   {
     case AZ_MQTT5_EVENT_CONNECT_RSP:
       ref_connack++;
+      az_mqtt5_connack_data* test_connack_data = (az_mqtt5_connack_data*)event.data;
+      assert_int_equal(test_connack_data->connack_reason, 0);
```

Comment:
Would we ever test a case that has a different connack reason?

Reply:
We could do so once we expand the tests, in this case we want to ensure that the connection is accepted